### ISSUE TITLE
Documents containing "œ" / "æ" not found when searching "oe" / "ae"

### ISSUE THREAD

**Issue by FlorianVenturini** [meilisearch/meilisearch#4405]
**Describe the bug**
I have an index with some documents having characters like `œ` or `æ` which don't show up when I type `oe` / `ae`

**To Reproduce**
1. Insert a very basic document containing the problematic character
```bash
curl \
  -s -X PUT 'http://localhost:7700/indexes/fishes/documents' \
  -H 'Content-Type: application/json' \
  --data-binary '[{ "id": "1", "name": "Cœlacanthe" }]'
```
2. Search this document using different queries
```bash
curl \
  -s -X POST 'http://localhost:7700/multi-search' \
  -H 'Content-Type: application/json' \
  --data-binary '{
    "queries": [
      { "indexUid": "fishes", "q": "coelac" },
      { "indexUid": "fishes", "q": "colac" },
      { "indexUid": "fishes", "q": "celac" },
      { "indexUid": "fishes", "q": "cœlac" }
    ]
  }'
```

**Expected behavior**
I expect each query to be successful, but instead only the last 3 return something 😕 
The first one (`coelac`) returns nothing

**Meilisearch version:**
`v1.6.1`

**Additional context**
Tested locally using docker image and in our self-hosted environment (DigitalOcean)

**Comment by ManyTheFish** [comment]
Hello @FlorianVenturini,

This issue is related to the tokenization,
I've created an issue on Charabia to solve it:
https://github.com/meilisearch/charabia/issues/268

Thank you for creating this issue,
see you!

**Comment by curquiza** [comment]
Hello everyone, this issue is fixed since [v1.9.0](https://github.com/meilisearch/meilisearch/releases/tag/v1.9.0) 🎉
